article/_posts/2022-04-14-MultiModalVersatileNetworks.md

```diff
@@ -0,0 +1,107 @@
+tags: deep-learning representation-learning self-supervised multi-task
+author: "Nathan Painchaud"
+cite:
+    authors: "Jean-Baptiste Alayrac, Adria Recasens, Rosalia Schneider, Relja Arandjelović, Jason Ramapuram, Jeffrey De Fauw, Lucas Smaira, Sander Dieleman, Andrew Zisserman"
+    title:   "Self-Supervised MultiModal Versatile Networks"
+    venue:   "NeurIPS 2020"
+pdf: "https://proceedings.neurips.cc/paper/2020/file/0060ef47b12160b9198302ebdb144dcf-Paper.pdf"
+---
+
+
+# Highlights
+- Introduce the notion of a MultiModal Versatile (MMV) network, that can ingest multiple modalities and outputs
+representations useful for downstream tasks in multiple modalities;
+    - The papers especially studies how best to combine the modalities, so that representations respect four properties
+- Introduce the process of deflation, to efficiently apply networks on video data to static images.
+
+
+# Introduction
+The authors' motivation is to propose a network for joint video-audio-text representation that respects four properties:
+1. Able to take as input any of the three modalities;
+2. Respect the specificity of the modalities: audio and video are much finer than text, i.e. their dimensionality is
+higher and they contain a lot of details, by contrast to the high-level, coarse information provided by text data;
+3. Enable the different modalities to be easily compared even when they are never seen together during training;
+4. Efficiently applicable to visual data coming in the form of dynamic videos or static images.
+
+
+# Methods
+A video $$x$$ can be described as a combination $${x_m}$$ of different modalities $$m \in M$$, e.g. vision $$x_v$$,
+audio $$x_a$$ and text $$x_t$$. For each modality $$x_m$$, the authors propose to learn a specific *backbone neural
+network* $$f_m$$ that outputs a *representation vector* of dimension $$d_m$$, such that
+$$f_m: \mathcal{X}_m \rightarrow \mathbb{R}^{d_m}$$. Finally, modality specific *projection heads*
+$$g_{m \rightarrow s}$$ map these representation vectors to vectors $$z_{m,s}$$ in a space $$S_s$$ shared between
+modalities. Thus, the final representation of an input modalities is:
+
+$$
+z_{m,s} = g_{m \rightarrow s}( f_m(x_m) )
+$$
+
+The authors' goal is to find the best way to combine the modality specific representation vectors into a shared space,
+a process which they represent by what they call *embedding graphs*. The figure below represents different embedding
+graphs discussed in the paper":
+
+![](/article/images/MultiModalVersatileNetworks/figure1.jpg)
+
+
+## MMV: MultiModal Versatile Networks
+Between the different embedding graphs proposed in Fig. 1, the authors settle on the Fine And Coarse spaces (FAC). They
+argue that a higher-dimensional video-audio shared space ($$S_{va}$$) is necessary, since these modalities contain finer
+details than the high-level, coarse audio modality. However, since another projection maps $$S_{va}$$ to the shared
```

Comment:
"coarse audio modality": "coarse text modality" ? You mentioned earlier that the coarse modality is the text, and you have just said "a higher-dimensional video-**audio** shared space ($$S_{va}$$) is necessary, since these modalities contain finer
details"

Reply:
My bad! This was a brain-fart, and I meant to say "coarse **text** modality" here.